doc/literate.md

```diff
@@ -0,0 +1,89 @@
+# Literate Specification
+
+We have developed a workflow for writing literate, executable formal specs using
+the tool [lmt](https://github.com/driusan/lmt). This allows us to embed the
+formal specification of a system within informal, natural language
+specifications, and to extract the formal parts into executable code. This code
+can then be used for testing or verifying the spec, or fed into a toolchain for
+model based testing.
+
+## Prerequisites
```

Comment:
Ah, python is there because of this:
https://github.com/informalsystems/quint/blob/main/tutorials/Makefile#L27

Perhaps we can just say something like "if you are using nix, look at this nix file which sets up `lmt`"

Reply:
Added that. Thanks :)

Reply:
Ah I missed this:

> Moreover, is quint really a dependency here?

It's a dependency for executing the code extracted, which is what the example in the remainder of the document shows.